content/_code-samples/escrow/js/create-escrow.js

```diff
@@ -1,57 +1,56 @@
 'use strict'
-const RippleAPI = require('ripple-lib').RippleAPI
-const cc = require('five-bells-condition')
-const crypto = require('crypto')
-
-const myAddr = 'rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn'
-const mySecret = 's████████████████████████████'
-
-// Construct condition and fulfillment
-const preimageData = crypto.randomBytes(32)
-const myFulfillment = new cc.PreimageSha256()
-myFulfillment.setPreimage(preimageData)
-const conditionHex = myFulfillment.getConditionBinary().toString('hex').toUpperCase()
-
-console.log('Condition:', conditionHex)
-console.log('Fulfillment:', myFulfillment.serializeBinary().toString('hex').toUpperCase())
-
-// Construct transaction
-const currentTime = new Date()
-const myEscrow = {
-  "destination": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn", // Destination can be same as source
-  "destinationTag": 2017,
-  "amount": "0.1113", //decimal XRP
-  "condition": conditionHex,
-  "allowExecuteAfter": currentTime.toISOString() // can be executed right away if the condition is met
-}
-const myInstructions = {
-  maxLedgerVersionOffset: 5
-}
+if (typeof module !== "undefined") {
+  // Use var here because const/let are block-scoped to the if statement.
+  var xrpl = require('xrpl');
+};
+
+const cc = require('five-bells-condition');
+const crypto = require('crypto');
+
+const secret = "sEdTPPEeMH6SAgpo6rSj8YW7a9vFfUj";
```

Comment:
In code samples, try to not use previously funded accounts that are tested on. Generating an account using the faucet feature is the best practice:
```js
    console.log("Requesting addresses from the Testnet faucet...")
    const { wallet } = await client.fundWallet()
```

Reply:
added